docs/adr/0004-withdraw-expiry.rst

```diff
@@ -0,0 +1,91 @@
+Withdraw & Expiry
+=================
+
+**Status**: proposed
```

Comment:
A bit bikeshedding but the withdraw is not proposed. It has already been implemented. I guess the decision is on the two presented options to handle the partner being unavailable and not on implementing withdrawing as a whole.

Reply:
It is implemented... but isn't "proposed" the final state? after which we decided to implement one option. Or is it not the case?

Reply:
There is no master ADR template afaik. But "proposed" does not sound final to me.